cp-connector/pkg/controlplane/controlplane.go

```diff
@@ -4,42 +4,52 @@ import (
 	"context"
 	"errors"
 	"fmt"
-
 	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/keptn/cp-connector/pkg/eventmatcher"
+	"github.com/keptn/keptn/cp-connector/pkg/eventsource"
+	"github.com/keptn/keptn/cp-connector/pkg/logforwarder"
 	"github.com/keptn/keptn/cp-connector/pkg/logger"
+	"github.com/keptn/keptn/cp-connector/pkg/subscriptionsource"
+	"github.com/keptn/keptn/cp-connector/pkg/types"
 )
 
+// EventSenderKey can be used to get the event sender from the context
+// Deprecated: use own value of EventSenderKeyType
+var EventSenderKey = EventSenderKeyType{}
```

Comment:
does it make sense to also remove this @warber ?

Reply:
absolutely :)